SparkPost/2web2ui
# FE-696 - Remove docs files stored in the repo, migrate to wiki

[FE-696](https://jira.int.messagesystems.com/browse/FE-696)

### What Changed
* Removed `.md` files from the `docs` directory in the repo
* Migrated content to the wiki
* Added Fast and Furious gif to Wiki footer to appear on every page
* Added inspirational quote from Dominic Toretto

### How To Test
* Look through the [wiki](https://github.com/SparkPost/2web2ui/wiki) and check for any missing content
* Verify duplicated content was deleted

### To Do
